Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md

```diff
@@ -137,241 +308,122 @@ The proxy intercepts HTTPS traffic, presents a public CA certificate, then proxi
+- The chat commands are only supported when using the `Authorization Code flow`.
+- Posting a message or adaptive card to a private/shared channel is currently not supported in the ***send-notification*** command. Thus, also the ***mirror_investigation*** command does not support private/shared channels. For more information, see [Microsoft General known issues and limitations](https://learn.microsoft.com/en-us/connectors/teams/#general-known-issues-and-limitations).
+- The ***send-notification*** command currently supports only adaptive cards without user responses. To collect user responses from adaptive cards, please use the ***MicrosoftTeamsAsk*** command.
+- In case of multiple chats/users sharing the same name, only one will be used.
+- If a non-Cortex XSOAR/XSIAM user ran the `new incident` command in the chat with the bot, the owner of the created incident would be the logged in Cortex XSOAR/XSIAM user, not the external user who ran the command.
+- See Microsoft documentation for [Limits and specifications for Microsoft Teams](https://learn.microsoft.com/en-us/microsoftteams/limits-specifications-teams).
 
-### Create the Demisto Bot in Microsoft Teams
+## Important Information
 
-#### Creating the Demisto Bot using Microsoft Azure Portal
+- This integration is supported in Cortex XSOAR 8 and up and Cortex XSIAM without using an engine.
+- The integration has the ability to run built-in Cortex XSOAR commands, through a mirrored channel. Make sure to pass the command in the chat exactly as typed in the CORTEX XSOAR CLI. For example: `!DeleteContext all=yes`. Use the command `mirror-investigation` to mirror/create a mirrored channel.
+- For use cases where it is only needed to send messages to a specific channel, we recommend checking the [Microsoft Teams via Webhook Integration](https://xsoar.pan.dev/docs/reference/integrations/microsoft-teams-via-webhook), which has a simpler setup.
+- The messaging endpoint must be one of the following:
+  - The URL of the Cortex XSOAR/XSIAM server, including the configured port
+  - The Cortex XSOAR/XSIAM rerouting URL that you've defined for your Microsoft Teams instance (see the [Using Cortex XSOAR or Cortex XSIAM rerouting](#1-using-cortex-xsoar-or-cortex-xsiam-rerouting) section for more details)
+  - A proxy that redirects the messages received from Teams to the Cortex XSOAR/XSIAM server (see the [Using NGINX as reverse proxy](#2-using-nginx-as-reverse-proxy) section for more details)
+- Microsoft Teams will send events to the messaging endpoints via HTTPS request, which means the messaging endpoint must be accessible for Microsoft Teams to reach to it. As follows, the messaging endpoint can not contain private IP address or any DNS that will block the request from Microsoft Teams.
```

Comment:
Microsoft Teams sends events to the messaging endpoints via an HTTPS request, so the messaging endpoint must be accessible for Microsoft Teams to reach it. As a result, the messaging endpoint cannot contain a private IP address or any DNS that will block the request from Microsoft Teams.